Automattic/woocommerce-payments
# Update/base plugin references

#### Changes proposed in this Pull Request

In https://github.com/Automattic/woocommerce-payments/pull/2975 we changed the way we refer to the Subscriptions core library. This PR updates 2 places that still used the _base_ nomenclature.  





#### Testing instructions

* I couldn't find any references to `get_subscriptions_core_version()` as far as I could tell this is a helper function introduced in case we need to know which version of the core library is running. 
* The `is_subscriptions_enabled()` function is used pretty widely. Incorrectly determining if subscriptions is active is likely to lead to a wide range of issues but the way I noticed this was in the UPE.
1. Enable UPE via **WCPay Dev Utils** (https://d.pr/i/zU9S1S)
2. Navigate to **Payments → Settings**
3. Scroll down the page to **"Payment methods"** and add the Sofort PM (https://d.pr/i/iyIhix)
4. Scroll down to the **Advanced settings** and enable multi-currency (https://d.pr/i/OeDgIH)
4. Save the changes
5. Add EUR currency by following this guide: https://docs.woocommerce.com/document/payments/currencies/multi-currency-setup/
6. Add a subscription product to your cart. 
7. Navigate to the checkout page
8. Change your currency to EUR (the other payment methods are only supported using Euros) https://d.pr/i/sKdAel. 
9. Scroll down to the payment methods section on checkout. 
    - On this branch, you should only see SEPA. (https://d.pr/i/w9Gbvw)
    - On `feature/stripe_billing` you will see other options (Giropay and Sofort). This is because WC Pay is incorrectly recognising that Subscriptions is active and isn't limiting the available payment methods. https://d.pr/i/sKdAel

-------------------


**Post merge**
